src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js

```diff
@@ -88,11 +90,28 @@ const InstructorAddCourseCards = props => {
     schema.isValid(classData).then(valid => setDisabled(!valid));
   }, [classData]);
 
+  function success() {
+    Modal.success({
```

Comment:
Nice. Clear and concise

Reply:
I agree. I've sent Paul a Slack message on how to fix this issue. I believe what we were doing by mistake was pulling each branch down and not realizing that it had cause a merging mess. I'm working on this with the other frontend engineers to untangle this mess.